src/SemanticLog/ResourceErrorContext.php

```
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BEAR\Resource\SemanticLog;
+
+use Koriym\SemanticLogger\AbstractContext;
+
+final class ResourceErrorContext extends AbstractContext
```

PHPStan (2.1.17)

Class BEAR\Resource\SemanticLog\ResourceErrorContext extends unknown class Koriym\SemanticLogger\AbstractContext.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)